src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -552,6 +552,19 @@ void TableDockWidget::deleteAll()
   }
 }
 
+void TableDockWidget::noPeakFound()
+{
+  QMessageBox *warning = new QMessageBox(this);
+
+    auto htmlText = QString("<p>No peaks found.</p>");
```

Comment:
"No peaks we detected. Please check if you settings are not too strict for your data or whether you are able to see any peaks while manually browsing through a compound database."

Try to include a suggestion for the user, unless it is totally obvious what went wrong. This dialog could potentially come up for a user who just started using the software and made a simple yet non-obvious mistake.